index.htm

```diff
@@ -0,0 +1,168 @@
+        <div class="form-group">
+          <label id="number-label" for="number"
+            >Age<span class="clue">(optional)</span></label
+          >
+          <input
+            type="number"
+            name="age"
+            id="number"
+            min="10"
+            max="99"
+            class="form-control"
+            placeholder="Age"
+          />
+        </div>
+        <div class="form-group">
+          <p>How many times have you had Biltong?</p>
+          <select id="dropdown" name="role" class="form-control" required>
+            <option disabled selected value>Select</option>
+            <option value="once">Once</option>
+            <option value="many">Many times</option>
+            <option value="never">Never</option>
+          </select>
+        </div>
+    
+        <div class="form-group">
+          <p>Would you recommend Biltong kingdom to a friend?</p>
+          <label>
+            <input
+              name="user-recommend"
+              value="definitely"
+              type="radio"
+              class="input-radio"
+              checked
+            />Definitely</label
+          >
+          <label>
+            <input
+              name="user-recommend"
+              value="maybe"
+              type="radio"
+              class="input-radio"
+            />Maybe</label
+          >
+    
+          <label
+            ><input
+              name="user-recommend"
+              value="not-sure"
+              type="radio"
+              class="input-radio"
+            />Not sure</label
+          >
+        </div>
+        <div class="form-group">
+          <p>
+            What is your favorite biltong flavor?
+          </p>
+          <select id="most-like" name="mostLike" class="form-control" required>
+            <option disabled selected value>Select an option</option>
+            <option value="Peri-peri">Peri-peri</option>
+            <option value="Mild">Mild</option>
+            <option value="Garlic">Garlic</option>
+            <option value="Lemon and herb">Lemon and herb</option>
+          </select>
+        </div>
+    
+        <div class="form-group">
+          <p>
+            What would you like with your Biltong?
+            <span class="clue">(Check all that apply)</span>
+          </p>
+    
+          <label
+            ><input
+              name="prefer"
+              value="Extra sauce"
+              type="checkbox"
+              class="input-checkbox"
+            />Extra sauce</label
+          >
+          <label>
+            <input
+              name="prefer"
+              value="Salad"
+              type="checkbox"
+              class="input-checkbox"
+            />Salad</label
+          >
+          <label
+            ><input
+              name="prefer"
+              value="Gin"
+              type="checkbox"
+              class="input-checkbox"
+            />Gin</label
+          >
+          <label
+            ><input
```

Comment:
Please follow the correct Html practice and write the tags in the above lines properly.
this will make your code look professional.